### PR TITLE
feat: Added Android 13 docs

### PR DESCRIPTION
**Pull request by breautek**
cordova-android 13 info was missing

Screenshot from 2024-07-24 21-57-02

Screenshot from 2024-07-24 21-57-09

closes #1355 (improperly attempts to address cordova-android@13 docs)

### CODE REVIEW

**Comment by Hermes-Augusto-Dev** [comment]
I was checking the documentation site and noticed that it hasn't been updated yet.
at least that's what appears here for me.
I hope I was able to help with something
Captura de tela 2024-07-25 001722

**Comment by breautek** [comment]
> I was checking the documentation site and noticed that it hasn't been updated yet. at least that's what appears here for me. I hope I was able to help with something Captura de tela 2024-07-25 001722

The PR isn't merged yet. Waiting for at least one other contributor for a review.

Once merged, it will take about 30 mins to appear on the /dev/ version of the site.

**Review by NiklasMerz — APPROVED** [review]
LGTM and matches https://cordova.apache.org/announcements/2024/05/23/cordova-android-13.0.0.html

**Pull request merged**
